tests/test_memoryfs.py

```diff
@@ -72,14 +72,13 @@ def test_copy_preserve_time(self):
         self.fs.makedir("bar")
         self.fs.touch("foo/file.txt")
 
-        namespaces = ("details", "modified")
-        src_info = self.fs.getinfo("foo/file.txt", namespaces)
+        src_info = self.fs.getmodified("foo/file.txt")
```

Comment:
This should be renamed to `src_datetime` since it's not an info anymore for clarity.